src/vm/dllimport.cpp

```diff
@@ -5890,6 +5890,16 @@ bool         NDirect::s_fSecureLoadLibrarySupported = false;
 #define TOLOWER(a) (((a) >= W('A') && (a) <= W('Z')) ? (W('a') + (a - W('A'))) : (a))
 #define TOHEX(a)   ((a)>=10 ? W('a')+(a)-10 : W('0')+(a))
 
+#ifdef FEATURE_PAL
+#define PLATFORM_SHARED_LIB_SUFFIX_A PAL_SHLIB_SUFFIX
+#define PLATFORM_SHARED_LIB_PREFIX_A PAL_SHLIB_PREFIX
+#else // !FEATURE_PAL
+#define PLATFORM_SHARED_LIB_SUFFIX_A ".dll"
+#define PLATFORM_SHARED_LIB_PREFIX_A ""
+#define PLATFORM_SHARED_LIB_SUFFIX_W W(".dll")
```

Comment:
Nit: I do not think we need to have a defines for these since they are used just once.